### DISCUSSION TITLE
εργαλείο προτυποποίησης

### DISCUSSION THREAD

**Discussion by ghost** [sdy60/syllabus#57] [Q&A] [answered]
@epidrome είναι απαραίτητο να χρησιμοποιήσουμε εργαλείο προτυποποίησης ή μπορούμε να φτιάξουμε html ή κάποιο on line εργαλείο που φτιάχνει html (τύπου wix) για να προτυποποιήσουμε;
ευχαριστώ

**Comment by epidrome** [comment, marked as answer]
όπως λέει η θεωρία (που είδαμε στις σχετικές διαφάνειες), τα διαδραστικά υποδείγματα μεγάλης πιστότητας για δοκιμές με τους τελικούς χρήστες μπορούν να είναι ότι ταιριάζει στην κάθε περίπτωση, από κανονικό κώδικα παραγωγής, μέχρι βίντεο-μοντάζ.

επομένως το ερώτημα δεν είναι αν κάποιο δεδομένο εργαλείο είναι γενικά κατάλληλο, αλλά ποια είναι τα χαρακτηριστικά εκείνα της διάδρασης που θέλουμε να αξιολογήσουμε και τι είναι εφικτό να δώσουμε στους χρήστες.
